.travis.yml

```diff
@@ -4,9 +4,10 @@ rvm:
 - 2.7.5
 - 3.0.3
 cache: bundler
-gemfile:
-  - gemfiles/activesupport_6.0.gemfile
-  - gemfiles/activesupport_6.1.gemfile
+env:
+  matrix:
+    - TEST_RAILS_VERSION=6.1
+    - TEST_RAILS_VERSION=6.0
```

Comment:
This should be part of the env key just below...otherwise that env key overwrites this one

Reply:
(or we can just remove that other "global" one